### DISCUSSION TITLE
[suggestion] Use different model for different tasks.

### DISCUSSION THREAD

**Discussion by ackvf** [JudiniLabs/code-gpt-docs#54] [Ideas]
I would like to use `text-davinci-003` for "Explain CodeGPT", Stack Overflow and general chat, 
`code-davinci-002` for `ctrl+shift+i` invocation and code generation,
and `code-davinci-edit-001` for refactorings.

**Comment by davila7** [comment]
@ackvf 

That's an amazing idea, I'm going to move it into a discussion so we can evaluate it with the whole community.
